GL #9 Member status

str opened the pull request:
- remove console.log calls that were making noice, making it harder to debug
- helper memberTypeToText() now get a profile as param, to easily debug the full profile
- improve ProfileCache.getPromise documentation
- fix when to call the updateMemberType when login in (this whas the actual problem)
- fixed bug when editing a profile, when switching to markdown editor, the form was empty
- improved spacing in profile view
- use i18m in profile type label
- improved installation documentation about npm vs yarn

piecesnbits commented:
thanks for the code cleanup. it broke something though...
![image](https://user-images.githubusercontent.com/44613132/55397230-0ca93c00-5546-11e9-9121-8289a33a2f6e.png)

str committed:
GL #9 Member status

- remove console.log calls that were making noice, making it harder to debug
- helper memberTypeToText() now get a profile as param, to easily debug the full profile
- improve ProfileCache.getPromise documentation
- fix when to call the updateMemberType when login in (this whas the actual problem)
- fixed bug when editing a profile, when switching to markdown editor, the form was empty
- improved spacing in profile view
- use i18m in profile type label
- improved installation documentation about npm vs yarn
